### ISSUE TITLE
Netflix no longer working?

### ISSUE THREAD

**Issue by trevorheisler** [boxeehacks/boxeehack#432]
Since I installed the Boxee Plus firmware two days ago, my Netflix is no longer working on my Boxee Box. Netflix works fine on my laptop and netbook. However, when I click on the app on the Boxee Box now, it just returns me to the Apps page. The Box is also no longer showing under my connected devices when I log into my Netflix account on my laptop. And I am not presented with an activation code when I try clicking on the app on the Box. What happened??? Everything else seems to work fine.

Trevor

**Comment by mxswe** [comment]
Have you tried to go to setting in the boxee and clear all history in the netflix app? Worked for me!

**Comment by trevorheisler** [comment]
I did try that, but it didn't work for me. The only thing that worked was a factory restore of the box, which removed the Boxee Plus firmware. Now Netflix works again. I'll try adding Boxee Plus again. Hopefully Netflix will work afterwards.

**Comment by boxeehacks** [comment]
It should work. I'm using it myself almost daily. Another good option is clearing the netflix cache (which also happens during factory restore). It might have been something related to that.

**Comment by arnyyy** [comment]
I had the exact same problems, since I installed boxeeplus and doing a factory restore does not help. Basically when I open the netflix app, the screen stays blank for a few minutes and then eventually there is a netflix screen that pops up saying it can not connect to netflix. I am able to stream movies fine with Navi-X and the watch TV shows, but netflix has not worked since adding boxeeplus. I upgraded to 1.4.5 but still no luck.

**Comment by boxeehacks** [comment]
Have you tried clearing the netflix cache?

**Comment by arnyyy** [comment]
Yes, cleared my netflix data from the advanced options. tried default settings, tried full factory restore tried installing and uninstalling boxeeplus a full times but no luck. 

One thing that I am noticing is that it looks like my wifi connection is dropping on the Boxee when attempting to access netflix after it can't connect for a why. I am able to connect on a number of other streaming devices in the house so netflix itself is working fine. 

I also tried to use the boxeeplus to uninstall it but still not able to use netflix. Not sure if any settings are kept when you run the uninstall?

I do recall that it worked when I first installed boxeeplus and then it stopped working a few day later. At a loss here.

**Comment by boxeehacks** [comment]
If you uninstall there should be no traced left of Boxee+Hacks. If it still doesn't work after uninstall then it looks like an issue with your connection. Are you in the US or outside using a VPN or DNS service?

**Comment by arnyyy** [comment]
I'm in Canada connecting directly to netflix canada without a VPN. It has been working perfectly for the past year and only stopped working after boxee plus was installed. I even tried my US VPN connection to see if that would help but still not good. I updated my router to use google's open DNS servers and tried OpenDNS servers as well and still no good. 

It seems that the netflix app is causing the boxee to drop the connection. as soon as I exit out of it the connection is restored. 

It's possible that it is just a coincidence that it stopped working after boxeeplus was installed, but that seems very odd. Also Netflix is working on about 8 other devices in the house.

**Comment by tkaravou** [comment]
I'm having a similar issue with Netflix, in Canada, running Boxee+Hacks. What's happening with mine is I get a red netflix screen that loads, the I get a greyed out screen with the Netflix banner in the back and the loading bars. Thing is, nothing ever happens. I've never signed in and I never get a prompt to even sign in.

I've cleared the netflix cache, uninstalled netflix, reset all boxee settings, did a factory restore, deactivated all devices on netflix, updated my boxee firmware and tried again.. same thing. Red screen, then grey screen, then nothing. The app doesn't crash, it just hangs. I've tried with two separate internet connections (DSL and Cable), same thing.

**Comment by arnyyy** [comment]
I found that my wireless signal was causing the issues with netflix. All other streaming worked fine on wireless through boxee, but netflix didn't. When I tried a direct cable connection it worked perfectly. Not sure why, but I'm accessing ethernet now and it is working.

**Comment by ThatServerGuy** [comment]
I have the same issue. I wiped and loaded a plain boxee OS and fixed it. Loaded boxeeplus again and it's not working again :(

**Comment by ThatServerGuy** [comment]
Putting on a vanilla boxee reload now and testing again to verify.

**Comment by ThatServerGuy** [comment]
Another thing I notice is it drops my wifi connection and reconnects once netflix times out.

**Comment by tkaravou** [comment]
I'm not sure what happened, but this weekend, I tried again and magically, I got the log in screen. I didn't do anything special. It works perfectly now. I'm thinking, maybe it's a Netflix issue?

**Comment by arnyyy** [comment]
@r3zurector I have the same problem "Another thing I notice is it drops my wifi connection and reconnects once netflix times out." No idea why.

**Comment by gpavlidi** [comment]
Same here.. Installed the hack, was working fine, and at some point Netflix, and all other apps would not launch. they were segfaulting and I was returned to the apps page. Full factory reset fixes it, but it's a pain...

**Comment by Golong** [comment]
Hi 

Is Boxee + Hack done? Or will there be anymore updates?

**Comment by humboldtjs** [comment]
@Golong, please don't spam the issue tracking system. If you want to know these kinds of things either send an email, ask in the blog comments, or if you must add a new issue. But don't post to completely unrelated issues. It's difficult enough as it is to manage this large complex project without having to filter this kind of stuff out.

To answer your question though; no it's not done. But you will have to be a bit more patient, as all the easy features have been done, so any new release will take a while longer than before.

**Comment by Golong** [comment]
Ok. Sorry .

-----Original Message-----

**Comment by humboldtjs** [comment]
No worries ;) - just wanted to let you know for future cases. Just know that I appreciate that people are really eager to get updates for this project; it helps me stay motivated, so thanks! :)

**Comment by Golong** [comment]
I forgot to say, You are doing a great job.

We all Thank You.

John Felicciardi
Mets Roll Off
311 Winding Road
Old Bethpage, NY  11804
Bus# 516-249-5124
Fax #516-293-6609

-----Original Message-----

**Comment by kvz** [comment]
Netflix errors out for me too with a `Unable to connect to Netflix. Please try again or visit: www.netflix.com/help`
- Try Again
- Exit

I'm not on VPN, I'm in Holland (we have Netflix now), I've cleared cache, was running 1.51 firmware, just installed boxeehack from github master, verified internet is working (disabled wireless, getting vimeo media via wire just fine, spotify is also working).

Not sure what I can do to debug / be more helpful? The Netflix message sure isn't.

**Edit:** Switched to google nameserver (8.8.8.8) on the boxee but that didn't help either.

**Edit2:** Looks like it's because boxee+hacks allowed me to see/use the Netflix app again, but unfortunately the app itself needs to be updated for newly whitelisted countries to be able to connect (the restriction is outdated, but hardcoded in the app), and unfortunately it doesn't look like D-Link is in a rush to do anything about that (http://jira.boxee.tv/browse/BOXEE-11389) as Boxee was aquired by Samsung and basically unmaintained.

**Edit3:** If you're willing to throw money at this problem ($4.99 a month), you could try a service like http://support.unblock-us.com/customer/portal/articles/291557 to fool the Netflix app into thinking you are connecting from the USA.
